.github/workflows/race.yml

```diff
@@ -13,5 +13,8 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v2
 
+    - name: Replace sync.RWMutux with deadlock.RWMutux
```

Comment:
Since the tests are the same for all workflows, I suggest to change this workflow (race.yml) to find the deadlocks situations. 
Let's remove the `replace_mutux` from the Makefile and add it here.

Reply:
line # 17: `find . -type f -name "*.go" -not -path '*/\.*' -exec sed -i -- 's/\t"sync"/\tsync "github.com\/sasha-s\/go-deadlock"/g' {} +`

Line #2: `Deadlock and data race detection`

Rename the file to `deadlock.yml`